tests/acceptance/features/apiGraph/fullSearch.feature

```diff
@@ -8,20 +8,58 @@ Feature: full text search
     Given user "Alice" has been created with default attributes and without skeleton files
 
 
-  Scenario Outline: search files using a tag
+  Scenario Outline: search files by tag
     Given using <dav-path-version> DAV path
-    And user "Alice" has uploaded file with content "hello world" to "file1.txt"
-    And user "Alice" has uploaded file with content "Namaste nepal" to "file2.txt"
-    And user "Alice" has uploaded file with content "hello nepal" to "file3.txt"
-    And user "Alice" has created the following tags for file "file1.txt" of the space "Personal":
+    And user "Alice" has created the following folders
+      | path                      |
+      | folderWithFile            |
+      | folderWithFile/subFolder/ |
+    And user "Alice" has uploaded the following files with content "some data"
+      | path                                             |
+      | test.txt                                         |
```

Comment:
waht about this?
```suggestion
      | fileInRootLevel.txt                                         |
```

Reply:
Of course, determining if a file is at the root level will be simple.